treasue hunt/treasurehunt.html

```diff
@@ -5,69 +5,14 @@
 	<title>Treasure Hunt</title>
 	<link rel="stylesheet" type="text/css" href="treasurehunt.css">
 </head>
-<body>
-	<table>
-		<tr>
-			<td class="cell" id="0"></td>
-			<td class="cell" id="1"></td>
-			<td class="cell" id="2"></td>
-			<td class="cell" id="3"></td>
-			<td class="cell" id="4"></td>
-			<td class="cell" id="5"></td>
-			<td class="cell" id="6"></td>
-			<td class="cell" id="7"></td>
-		</tr>
-		<tr>
-			<td class="cell" id="8"></td>
-			<td class="cell" id="9"></td>
-			<td class="cell" id="10"></td>
-			<td class="cell" id="11"></td>
-			<td class="cell" id="12"></td>
-			<td class="cell" id="13"></td>
-			<td class="cell" id="14"></td>
-			<td class="cell" id="15"></td>
-		</tr>
-		<tr>
-			<td class="cell" id="16"></td>
-			<td class="cell" id="17"></td>
-			<td class="cell" id="18"></td>
-			<td class="cell" id="19"></td>
-			<td class="cell" id="20"></td>
-			<td class="cell" id="21"></td>
-			<td class="cell" id="22"></td>
-			<td class="cell" id="23"></td>
-		</tr>
-		<tr>
-			<td class="cell" id="24"></td>
-			<td class="cell" id="25"></td>
-			<td class="cell" id="26"></td>
-			<td class="cell" id="27"></td>
-			<td class="cell" id="28"></td>
-			<td class="cell" id="29"></td>
-			<td class="cell" id="30"></td>
-			<td class="cell" id="31"></td>
-		</tr>
-		<tr>
-			<td class="cell" id="32"></td>
-			<td class="cell" id="33"></td>
-			<td class="cell" id="34"></td>
-			<td class="cell" id="35"></td>
-			<td class="cell" id="36"></td>
-			<td class="cell" id="37"></td>
-			<td class="cell" id="38"></td>
-			<td class="cell" id="39"></td>
-		</tr>
-		<tr>
-			<td class="cell" id="40"></td>
-			<td class="cell" id="41"></td>
-			<td class="cell" id="42"></td>
-			<td class="cell" id="43"></td>
-			<td class="cell" id="44"></td>
-			<td class="cell" id="45"></td>
-			<td class="cell" id="46"></td>
-			<td class="cell" id="47"></td>
-		</tr>
+<body class="container">
+	<h2>Treasure Hunt</h2>
+	<h3>use arrow keys or (u,d,l,r) to move the player</h3>
+	<h4>collect all the treasures to win and use the swords to kill the monsters, if you don't have a sword you can use a potion to recover and if you don't have either of them</h4>

```

Comment:
Collect all the treasures to win! Beware of monsters that will try to stop you from collecting your treasure. You can use swords to vanquish them or use potions to recover any damage sustained from them but it'll be game over if you run out!